cli/src/commands/deploy.ts

```diff
@@ -117,9 +134,38 @@ export default class Deploy extends Command {
     return digest;
   }
 
+  public async selectApplicationForProject(): Promise<string | undefined> {
+    const NEW_APPLICATION = '__new_application__';
```

Comment:
consider using something unique (though getting __new_application__ as the id is very unlikely currently)